dashboard/lib/policies/script_activity.rb

```diff
@@ -0,0 +1,25 @@
+class Policies::ScriptActivity
+  def self.completed?(user, script)
+    return false unless user
+    return false unless script
+    user_script = user.user_scripts.find_by(script: script)
+    return false unless user_script
+    !!user_script.completed_at || user.completed_progression_levels?(script)
+  end
+
+  def self.a_level_passed?(user, script)
```

Comment:
Naming nit (especially since this is more-or-less a private helper): I really wish this read something like `passed_any_level?` to mirror Ruby's `any?`